src/identity.ts

```diff
@@ -0,0 +1,135 @@
+import * as superagent from 'superagent';
+
+import { getAuthedProfile } from './profile';
+
+const USER_KEY = 'fusebit.tenantId';
+
+interface IEveryAuthCredential {
+  id: string;
+  tags: IEveryAuthTagSet;
+  dateModified: string;
+  access_token?: string;
```

Comment:
Re: https://5qtrs.slack.com/archives/CE08851PE/p1649362916477679, let's normalize to `accessToken`